src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -1142,6 +1142,12 @@ void FieldProps::apply_tran(const std::string& keyword, std::vector<double>& dat
     Opm::apply_tran(this->tran, this->double_data, this->active_size, keyword, data);
 }
 
+void FieldProps::drop_tran(const std::string& keyword) {
+    auto& tran = this->tran.at(keyword);
```

Comment:
I think you should choose another name for the local here.  Jenkins reports:
```
/var/lib/jenkins/workspace/opm-common-PR-builder/src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp: In member function 'void Opm::FieldProps::drop_tran(const string&)':
/var/lib/jenkins/workspace/opm-common-PR-builder/src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp:1146:11: warning: declaration of 'tran' shadows a member of 'Opm::FieldProps' [-Wshadow]
     auto& tran = this->tran.at(keyword);
           ^~~~
In file included from /var/lib/jenkins/workspace/opm-common-PR-builder/src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp:40:
/var/lib/jenkins/workspace/opm-common-PR-builder/src/opm/parser/eclipse/EclipseState/Grid/FieldProps.hpp:512:65: note: shadowed declaration is here
     std::unordered_map<std::string, Fieldprops::TranCalculator> tran;
                                                                 ^~~~
```

Reply:
Good point - I will!